docs/qcforward/bw_vs_gridprops.inc

```diff
@@ -0,0 +1,183 @@
+given or a full stop of the workflow is forced.
+
+The input to this method is a python dictionary with some defined keys. Note that
+the order of keys does not matter. It is advised to study the examples further below.
+
+
+Common fields (same input inside or outside RMS)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+verbosity
+  Level of output while running None, "info" or "debug", default is None (optional).
+
+compare:
+  A dictionary on the form ``{"Facies": "FACIES", "PHIT": "POROSITY"}`` where the first
+  elements are the property name in blocked wells (here ``"Facies, "PHIT"``) and the second
+  elements (after the colon) are the corresponding name in the grid properties (here
+  ``"FACIES", "POROSITY"``). Note that names can of course be equal.
+
+
+actions
+  This is a list of dictionaries that shows what actions which shall be performed per
+  well (any) or for the average (all), for example ``{"warn": "all < 50", "stop": "all <
+  30"}`` which means that match < than 50% will trigger a warning, while a match < 30%
+  will trigger a stop in work flow. Note that ``any`` and ``all`` shall not be mixed in
+  the same record (required). Also both lines are (currently) required.
+
+report
+  Result will be written in a CSV file (which e.g. can be used in plotting) on disk.
+  (optional).
+
+dump_yaml
+  If present, should be a file name where the current data structure is dumped to YAML
+  format. Later this YAML file can be edited and applied for a single line input
+  (optional).
+
+nametag
+  A string to identify the data set. Recommended.
+
+show_data
+  A string (or bool) if the full dataset shall be shown to screen as a dataframe
+  (table). This can be useful when investigating what wells that triggers e.g. a stop.
+  Possible values are False (default, means no output), "warn" to wells with WARN or
+  STOP criteria, "fail" to see wells that fails only (due to STOP) and True or "all" to
+  show everything. Any other words (like "foo") will be interpreted as "all").
+
+
+Keys if run inside RMS
+^^^^^^^^^^^^^^^^^^^^^^
+
+bwells
+  In RMS this is a dictionary with 3 fields: ``names``, ``grid`` and ``bwname``.
+  The names is a list of wellnames which in turn can have python valid regular
+  expressions. See examples. (required).
+
+grid
+  Name of grid icon in RMS (required).
+
+
+If run in normal python (terminal or ERT job)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+bwells
+  Outside RMS, wells is a list of files on RMS ascii well format. File wildcards are
+  allowed, se example. (required).
+
+grid
+  Name of file with grid (in ROFF, EGRID or GRDECL format) (required).
+
+gridprops
+  A list of list where the inner list is a pair with name of Zone and assosiated
+  filename, for example ``[["Zone", "zone.roff"]]``. This is required when running
+  outside RMS but not needed when run inside RMS.
+
+
+Example when run inside RMS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: python
+
+    import pathlib
```

Comment:
Change to `from pathlib import Path`
To avoid NameError saying 'Path' is not defined